frontend/src/components/auth/Login.tsx

```diff
@@ -42,17 +42,17 @@ const Login = (): React.ReactElement => {
   const isVerified = data?.isVerified;
 
   const onLogInClick = async () => {
-    if (!isVerified) {
-      window.alert(
-        "Failed to log in. Please check your email for a link to verify your account.",
-      );
-    }
     const user: AuthenticatedUser = await authAPIClient.login(
       email,
       password,
       login,
     );
     setAuthenticatedUser(user);
+    if (isVerified === false) {
+      window.alert(
+        "Failed to log in. Please check your email for a link to verify your account.",
+      );
+    }
```

Comment:
This change is unrelated to the ticket... you might've noticed there's sometimes a little glitch with this alert popping up even when you log in with a verified account - the state of this component's a bit out of sync, so I was trying to move the alert down to fix it... but that didn't work lol we can make a bug ticket for this at some point

Reply:
The issue can be resolved by adding an await statement before isVerified. Also, please check if there are any other places with the same issue. 🙏 
```
if ((await isVerified) === false) {
      // eslint-disable-next-line no-alert
      window.alert(
        "Failed to log in. Please check your email for a link to verify your account.",
      );
}